hugo/layouts/langium/showcase-page.html

```diff
@@ -7,18 +7,58 @@
         height: 100%;
     }
 
+    .container {
+        width: 100%;
+        height: 100%;
+        border: 1px rgba(38, 136, 140) solid;
+        justify-content: center;
+        align-self: center;
+        display: flex;
+    }
+
+    .editor {
```

Comment:
Just a thought, on smaller screen sizes, like a phone, it's very difficult to fit both the editor & the preview side-by-side. I would recommend either using a media-query to break down these elements into one on top of the other, or to add some sort of 'tab' mechanism at the top of the page that allows switching between the two displays on smaller screens. The first approach would work and would be quicker, the second would take more time, and is maybe something we can consider later on.

Reply:
Here's an example.
![squish](https://user-images.githubusercontent.com/5070304/195791832-49f83669-6ad3-411c-b90d-723d299f2eb2.jpg)